pkg/networkservice/chains/endpoint/server.go

```diff
@@ -83,7 +81,7 @@ func (e *endpoint) Register(s *grpc.Server) {
 
 // Serve  - serves passed Endpoint on grpc
 func Serve(ctx context.Context, listenOn *url.URL, endpoint Endpoint, opt ...grpc.ServerOption) <-chan error {
-	server := grpc.NewServer(append(spanhelper.WithTracing(), opt...)...)
+	server := grpc.NewServer(opt...)
```

Comment:
Do we need also do the same change for the client?

Reply:
In #533 I've added options to `sandbox.NewClient()`, which we use only for tests. Did you meant it?

Reply:
yes, you're correct